### PR TITLE
 Add user password reset 

### PR DESCRIPTION
**Pull request by jaimeferreira11**
Add page to change password

### CODE REVIEW

File: src/pages/recovery-password.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/* global jQuery */
```
Comment (rodrigojv) [inline src/pages/recovery-password.js]: A better name for this file could be "reset-password.js"

File: src/pages/recovery-password.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+var API_PUBLIC =  "http://127.0.0.1:8080";
```
Comment (rodrigojv) [inline src/pages/recovery-password.js]: This should not be hardcoded, we need to come up a better way get the API url in this place.

File: src/pages/recovery-password.js (hunk 1 above)
Comment (rodrigojv) [reply src/pages/recovery-password.js]: I think a better approach is moving this JS code to a backbone view in `login_app/reset/view.js`,

Then we can include login_main.js in the recovery.html and finally access the reset view by way of a backbone route: `login/reset`.

When the user goes to `login/` we can show the login/view.

**Comment by jaimeferreira11** [comment]
Would you please check it again?

**Comment by rodrigojv** [comment]
Can't access the reset page from login, I think we should do something like this:
https://stackoverflow.com/questions/21412714/backbone-routing-with-static-html-files

So that we can acces the reset page by visiting: http://localhost:9000/login/index.html#reset

I tried adding the line:
```javascript
 Bn.history.start({ root: window.location.pathname });
```
And it worked for me.

**Comment by rodrigojv** [comment]
@jaimeferreira11 We can also delete API_PUBLIC in env.json.

**Comment by jaimeferreira11** [comment]
API_PUBLIC is used in login_app/reset/view.js

**Pull request merged**